app/query/QueryStringHelper.scala

```diff
@@ -0,0 +1,35 @@
+package query
+
+import models.RatingInfosRequest
+
+import scala.util.Try
+
+object QueryStringHelper {
+  def parseQueryParams(queryParams: String): RatingInfosRequest = {
+    queryParams.replace("?", "")
+    val maybeParamsStr = Try(queryParams.split("&")).toOption
+    maybeParamsStr.map { paramStr =>
+      val maybeMovieIdsStr = paramStr.headOption
+      val maybeMovieIds = maybeMovieIdsStr.flatMap { movieIdsStr =>
+        val maybeMovieIdsParams = Try(movieIdsStr.split("=")).toOption
+        maybeMovieIdsParams.map{movieIdsParams =>
+          val movieIdsStr = movieIdsParams.headOption
+          if (movieIdsStr.exists(paramName => paramName.equals("movieIds"))) {
+            val maybeMovieIds = Try(movieIdsParams(1).split(",")).toOption.map(_.map(_.toInt))
+            val maybeLimitParam = Try(paramStr(1)).toOption
+            maybeLimitParam.map { limitParam =>
+              val maybeLimitParams = Try(limitParam.split('=')).toOption
+              maybeLimitParams.flatMap(limitParams => limitParams.headOption)
+            }
+          }
+
+        }
+      }
+//        Try(movieIdsStr.split(",").map(_.toInt)).toOption)
```

Comment:
remove commented line

Reply:
Will do, thanks!

Reply:
This is obsolete, no longer present in the last version.